web_widget_table/static/js/web_widget_table.js

```diff
@@ -0,0 +1,56 @@
+odoo.define('web_widget_table', function(require)
+{
+    var registry = require('web.field_registry'),
+        AbstractField = require('web.AbstractField'),
+        FieldDataTable = AbstractField.extend({
+        className: 'oe_form_field_table',
+        supportedFieldTypes: ['char'],
+        init: function()
+        {
+            this._super.apply(this, arguments);
+        },
+        _render: function()
+        {
+            this.$el.empty(); 
+            var field = this.nodeOptions['field_name'],
+                fieldModel = this.record.data[field].model,
+                fieldRecordIds = this.record.data[field].res_ids,
+                table = `<table><thead><tr>`,
+                table_fields = this.nodeOptions['fields'],
+                headers = this.nodeOptions['headers'];
+                datatable_params = this.nodeOptions['datatable_params'];
+
+            headers.forEach(element => {
+                table += '<th>' + element + '</th>';
+            });
+            table+= `</tr></thead><tbody>`;
+            this._rpc({
+                model: fieldModel,
+                method: 'read',
+                args: [fieldRecordIds],
+                })
+                .then(function(result){
+                    Array.from(result).forEach(record => {
+                        table+='<tr>'
```

Comment:
Semicolon missed.
To prevent these type of "mistakes" you can use eslint with the OCA rules: https://raw.githubusercontent.com/OCA/pylint-odoo/master/pylint_odoo/examples/.jslintrc

Here can found a guide to setup Atom editor: https://gist.github.com/Tardo/52a08cb7f98ae3cbfafdf47203c29a4b